docs/contributing.md

```diff
@@ -31,6 +31,14 @@ Or, if you are using Visual Studio:
 
 Make sure you [build the repo](#build-the-repo) from command line at least once. Then use `./start-code.sh` (macOS and Linux) or `.\start-code.cmd` to start VS Code.
 
+## Native build
+
+The default build includes native builds for `Aspire.Cli` which produces Native AOT binaries for some platforms. These projects are in `eng/clipack/Aspire.Cli.*`.
+
+By default it builds the cli native project for the current Runtime Identifier. A specific RIDs can be specified too by setting `$(TargetRids)` to a colon separated list like `$(TargetRids) = osx-x64:osx-arm64`.
```

Comment:
Why colon instead of semi-colon?

Reply:
To make it easier to pass values on the command line. `dotnet/runtime` uses `,`.

https://github.com/dotnet/runtime/blob/96e283d3938c40dd7eb3e8bb6c25827e66c45a26/src/libraries/sendtohelix.proj#L77-L78